src/action/interactive.rs

```diff
@@ -275,7 +275,6 @@ impl UserPicked {
                 KeyCode::Char('?') => {
                     println!("{}", HELP);
                     return self.user_input(suggestion, running_idx);
```

Comment:
don't do recursion, we might have a lot of them here, there must be a better way achieving this

Reply:
Only other option I coudl think of was a loop...